package.json

```diff
@@ -10,8 +10,9 @@
   "scripts": {
     "build": "babel src --out-dir lib --ignore test.js,test.jsx --source-maps",
     "coveralls": "cat coverage/lcov.info | coveralls",
+    "flow": "flow",
     "lint": "eslint . --ext .js,.jsx --fix",
-    "test": "yarn run test:unit && yarn run test:integration",
+    "test": "yarn run flow && yarn run test:unit && yarn run test:integration",
```

Comment:
We ran into some issues in running flow Travis in pro-web when adding new packages to node_modules (something caching related I think), so we had to disable flow in Travis for the time being till we can figure out what the deal with that is. Just a heads up in case you run into a similar issue!